rcv/find_line.py

```diff
@@ -89,9 +88,15 @@ def find_line_contour(image):
         box_area = cv2.contourArea(box)
         # find whether the current rectangle to contour ratio
         # is smaller than our current best
-        if box_area / contour_area < min_ratio:
+        M = cv2.moments(approx)
+        # calculate x,y coordinate of center
+        if M["m00"] != 0:
+            cY = int(M["m01"] / M["m00"])
+        else:
+            cY = 0
+        if min_ratio < cY:

```

Comment:
I thought that we are looking for the max cY where cY < 240 or 480 (depends on the resolution) ?